Apps/WebClient/src/ClientApp/src/utility/masks.ts

```diff
@@ -0,0 +1,14 @@
+export type MaskCharacter = string | RegExp;
+export type Mask = string | ((value: string) => MaskCharacter[]);
+
+export function zipCodeMask(value: string): MaskCharacter[] {
```

Comment:
American zip code mask is unlikely to be needed on the client app and postal code is not needed at this time.